yb-voyager/src/srcdb/pg_dump_extract_schema.go

```diff
@@ -95,6 +95,8 @@ func parseSchemaFile(exportDir string) {
 				objSqlStmts["TABLE"].WriteString(sqlStatement)
 			case "MATERIALIZED VIEW":
 				objSqlStmts["MVIEW"].WriteString(sqlStatement)
+			case "COLLATION":
```

Comment:
No, it can be seen that `COLLATION` is now included within `SCHEMA`, I have additionally tested that the collation related statements arrive after the schema related ones